code/modules/research/techweb/nodes/robo_nodes.dm

```diff
@@ -68,7 +68,7 @@
 
 /datum/techweb_node/ai_laws
 	id = TECHWEB_NODE_AI_LAWS
-	display_name = "Advanced AI Laws"
+	display_name = "Advanced AI Upgrades"
```

Comment:
Everything but one is still law, wouldnt it make more sense to have this be a separate node?

Reply:
Having a entirely new tech node for one middling upgrade would pretty much ensure that it never gets researched. If people add more upgrade disks then it would be best to move it into a new techweb, but with only one disk I don't see the point of doing so.